tests/goroutine_test.go

```diff
@@ -265,3 +266,18 @@ func TestEventLoopStarvation(t *testing.T) {
 	}()
 	<-ctx.Done()
 }
+
+func TestGoroutineBuiltin(t *testing.T) {
+	go print("")
+}
```

Comment:
What does this test? I suppose a failure crashes? Might be worth commenting to that effect for clarity?

Reply:
Added a comment and also improved the test to be more conclusive.